### PR TITLE
Multicluster - Add cluster params for metrics 

### PR DESCRIPTION
**Pull request by josunect**
Fix https://github.com/kiali/kiali/issues/6065

### CODE REVIEW

**Comment by josunect** [comment]
Param is not handle in the backend

**Comment by josunect** [comment]
When we are in multi cluster, send the cluster parameter in the query:

image

**Comment by josunect** [comment]
That should include the cluster as part of the metrics query: 

image

image

**Comment by hhovsepy** [comment]
@josunect can you please add which areas are touched, so to be tested?
Because in Overview page, the metrics queries do not have cluster param:
Screenshot from 2023-05-16 16-19-50

**Review by hhovsepy — DISMISSED** [review]
Looks good, @josunect thank you for the changes, tested locally.

File: frontend/src/pages/Graph/SummaryPanelCommon.tsx [hunk 1]
```diff
@@ -121,7 +121,7 @@ export const getNodeMetrics = (
     case NodeMetricType.APP:
       return API.getAppMetrics(nodeData.namespace, nodeData.app!, options, nodeData.cluster);
     case NodeMetricType.SERVICE:
-      return API.getServiceMetrics(nodeData.namespace, nodeData.service!, options, nodeData.cluster);
+      return API.getServiceMetrics(nodeData.namespace, nodeData.service!, options);
```
Comment (jshaughn) [inline frontend/src/pages/Graph/SummaryPanelCommon.tsx:124]: Are you sure we want to drop the 'cluster' here?  When you have selected a service node it is within a cluster (and within a namespace).  I think we may need to keep this discriminator.

File: frontend/src/services/Api.ts [hunk 2]
```diff
@@ -771,12 +771,8 @@ export const getIstioPermissions = (namespaces: string[], cluster?: string) => {
...
 };
 
-export const getMetricsStats = (queries: MetricsStatsQuery[], cluster?: string) => {
-  const queryParams: any = {};
-  if (cluster) {
-    queryParams.cluster = cluster;
-  }
-  return newRequest<MetricsStatsResult>(HTTP_VERBS.POST, urls.metricsStats, queryParams, { queries: queries });
+export const getMetricsStats = (queries: MetricsStatsQuery[]) => {
+  return newRequest<MetricsStatsResult>(HTTP_VERBS.POST, urls.metricsStats, {}, { queries: queries });
```
Comment (jshaughn) [inline frontend/src/services/Api.ts:775]: +1 on this change.

**Review by jshaughn — CHANGES_REQUESTED** [review]
Initially I thought there was a missing 'cluster' param for the getNamespaceMetrics requests from the full-graph side-panel.  But after thinking about I think it's correct, the full-graph traffic totals ahousl not narrow to a single cluster on those queries, they should show total traffic across clusters, for the namespace (note, this is only relevant for single-namespace graphs, we don't make those queries for multi-namespace graphs).

But, I don't think the same applies to the NamespaceBox side panel.  A NamespaceBox is constrained to a parent cluster, and as such I think the 'cluster' param needs to be added (see SummaryPanelNamespaceBox:553)

**Comment by matejnesuta** [comment]
Hello. Is this something that could use some back-end testing?

**Pull request merged**